projects/packages/my-jetpack/_inc/components/product-detail-table/index.jsx

```diff
@@ -114,6 +118,7 @@ const ProductDetailTableColumn = ( {
 		siteSuffix,
 		useBlogIdSuffix: true,
 		quantity,
+		adminUrl,
```

Comment:
The missing adminUrl was breaking the checkout flow. Not sure how long it's been that way

Reply:
Oh! Interesting.. Excellent catch and fix!! 🙌